README.md

```diff
@@ -2,11 +2,33 @@
 
 This project defines the keycloak theme shown on RenkuLab.
 
-# Production
+## Production
 
 In production, the Docker image is mounted into container running Keycloak. See the keycloak section of the helm chart/values file: https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/values.yaml. Search for `theme` to find the relevant lines.
 
-# Development
+## Configuration
+
+The theme can be configured to prompt a user to accept the **terms of service** on their first login. This is off by default, but to enable it, modify the following section in `renku-realm.json`:
```

Comment:
Note: Isn't this configurable from the Keycloak console as well?

Reply:
Yes, it is, but then an admin would need to check the configuration in the console every time Renku is deployed. If this is set in the file, then the setting will persist.